proxy/src/proxy.rs

```diff
@@ -455,6 +455,9 @@ impl<'a, S> Client<'a, S> {
 
 impl<S: AsyncRead + AsyncWrite + Unpin> Client<'_, S> {
     /// Let the client authenticate and connect to the designated compute node.
+    // Instrumentation logs endpoint name everywhere. Doesn't work for link
+    // auth; strictly speaking we don't know endpoint name in its case.
```

Comment:
With link auth, you could set the endpoint name later, after we know the endpoint.

(not sure it's worth the effort right now)

Reply:
I looked at the code, seems like currently we know only host/port `DatabaseInfo`, and I'm not sure how exactly host looks like, so yes, next time.